tizen/src/extractor.h

```diff
@@ -0,0 +1,153 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_EXTRACTOR_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_EXTRACTOR_H_
+
+#include <flutter/encodable_value.h>
+#include <flutter/method_call.h>
+#include <flutter/method_channel.h>
```

Comment:
`#include <flutter/method_channel.h>` is need?

Reply:
removed it